src/sentry/static/sentry/app/components/group/seenInfo.tsx

```diff
@@ -92,66 +68,78 @@ class SeenInfo extends React.Component<Props> {
-              releaseVersion={release.version}
-            >
-              <span>
-                <Version version={release.version} projectId={projectId} />
-              </span>
-            </VersionHoverCard>
-          </React.Fragment>
-        ) : !this.props.hasRelease ? (
-          <React.Fragment>
-            <NotConfigured>
-              <a href={this.getReleaseTrackingUrl()}>{t('Releases not configured')}</a>
-            </NotConfigured>
-          </React.Fragment>
-        ) : (
-          <React.Fragment>{t('Release n/a')}</React.Fragment>
-        )}
-        <StyledDateTime date={date} seconds />
-      </DateWrapper>
+      <HovercardWrapper>
+        <StyledHovercard
+          header={
+            <div>
+              <TimeSinceWrapper>
+                {t('Any Environment')}
+                <TimeSince date={dateGlobal} disabledAbsoluteTooltip />
+              </TimeSinceWrapper>
+              {environment && (
+                <TimeSinceWrapper>
+                  {toTitleCase(environment)}
+                  <TimeSince date={date} disabledAbsoluteTooltip />
+                </TimeSinceWrapper>
+              )}
+            </div>
+          }
+          body={<StyledDateTime date={date} />}
+          position="top"
+          tipColor={theme.gray500}
+        >
+          <DateWrapper>
+            {date ? (
+              <TooltipWrapper>
+                <StyledTimeSince date={date} disabledAbsoluteTooltip />
+              </TooltipWrapper>
+            ) : dateGlobal && environment === '' ? (
+              <React.Fragment>
+                <TimeSince date={dateGlobal} disabledAbsoluteTooltip />
+                <StyledTimeSince date={dateGlobal} disabledAbsoluteTooltip />
+              </React.Fragment>
+            ) : (
+              <React.Fragment>{t('n/a')} </React.Fragment>
+            )}
+          </DateWrapper>
+        </StyledHovercard>
+        <DateWrapper>
+          {defined(release) ? (
+            <React.Fragment>
+              {t('in release ')}
+              <VersionHoverCard
+                orgSlug={orgSlug}
+                projectSlug={projectSlug}
+                releaseVersion={release.version}
+              >
+                <span>
+                  <Version version={release.version} projectId={projectId} />
+                </span>
+              </VersionHoverCard>
+            </React.Fragment>
+          ) : null}
+        </DateWrapper>
+      </HovercardWrapper>
     );
   }
 }
 
-const NotConfigured = styled('span')`
-  margin-left: ${space(0.25)};
-`;
-
-const StyledDateTime = styled(DateTime)`
-  display: block;
-  font-size: ${p => p.theme.fontSizeSmall};
-  color: ${p => p.theme.gray300};
+const HovercardWrapper = styled('div')`
+  display: flex;
 `;
 
 const DateWrapper = styled('div')`
   margin-bottom: ${space(2)};
   ${overflowEllipsis};
 `;
 
+const StyledDateTime = styled(DateTime)`
+  color: ${theme.gray300};
+  font-size: ${theme.fontSizeMedium};
```

Comment:
In styled components, we should always use the theme from props and not from imports. The reason why is because if we were to change the theme due to dark mode, these colors would update where as if you use the imported theme, it won't.

(Even in this case where we want to always use `gray300`, we should use it from props to be consistent).